src/routes.rs

```diff
@@ -52,7 +52,7 @@ pub async fn get_xyz(super_collection: &str, parent: Uuid, collection: &str) ->
         entities
             .filter(ty.eq(collection))
             .filter(parent_id.eq(parent))
-            .select(id)
+            .select(Entity::as_select())
```

Comment:
What exactly does this mean? I grasp the idea but what does this expand to in terms of computation?

Reply:
It means "select units of the `Entity` structure", basically. Instead of selecting one field, it selects every field in that structure.

Reply:
Is it a compiletime expansion? Or is there runtime cost?